tests/x11/seahorse.pm

```diff
@@ -27,6 +27,10 @@ sub run {
     assert_screen "seahorse-name-new-keyring";        # Dialog  "Add a password keyring; name it"
     type_string "Default Keyring";                    # Name of the keyring
     send_key "alt-o";                                 # &Ok
+    unless (check_screen "seahorse-password-dialog") {
```

Comment:
I don't think this will work as expected as the `check_screen` has a timeout of 0 seconds. That could mean that he previous key press might cause caused an action already or not yet.